### PR TITLE
Added a tip about routes and container parameters

### PR DESCRIPTION
**Pull request by javiereguiluz**
This replaces #6505.

### CODE REVIEW

**Comment by xabbuh** [comment]
:+1:

File: book/routing.rst [hunk 1]
```diff
@@ -808,6 +808,13 @@ Path     Parameters
 ``/es``  *won't match this route*
 =======  ========================
 
+.. tip::
+
+    The route requirements can also include container parameters, as explained
+    in :doc:`this article </cookbook/routing/service_container_parameters>`.
+    This comes in handy when the regular expression is very complex and used
+    time and again in your application.
```
Comment (WouterJ) [inline book/routing.rst]: "is very complex and used time and again [...]"? I don't really understand what you're trying to tell with that bit.

Comment (javiereguiluz) [inline book/routing.rst]: "time and again" is an idiom for "repeatedly". In the example of the UUID, the problem is that the regexp is horrible. But in other cases the regexp is simple ... but used in 100 different routes ... so it's convenient to turn it into a parameter. Should I reword this?

Comment (WouterJ) [inline book/routing.rst]: thanks for the info. I've learned a new idiom now.

**Comment by WouterJ** [comment]
Thank you Javier.

**Pull request closed without merging**